tests/vm-prep

```diff
@@ -47,6 +47,9 @@ silent()
 
 prepare()
 {
+    # Make sure permissions are correct in containers
+    chown 666 /dev/kvm
```

Comment:
That won't survive a reboot and /lib/udev/rules.d/80-kvm.rules already sets it up like that, but it can't hurt either.